lib/ansible/plugins/connection/podman.py

```diff
@@ -98,6 +104,8 @@ def _connect(self):
         """
         super(Connection, self)._connect()
         rc, self._mount_point, stderr = self._podman("mount")
+        if rc != 0:
+            display.v("Failed to mount container %s: %s" % (self._container_id, stderr.strip()))
```

Comment:
the two lines below should be an `else` branch

Reply:
ack, fixed